src/NzbDrone.Core/Notifications/Notifiarr/NotifiarrProxy.cs

```diff
@@ -48,30 +43,25 @@ public ValidationFailure Test(NotifiarrSettings settings)
                 SendNotification(variables, settings);
                 return null;
             }
-            catch (HttpException ex)
+            catch (NotifiarrException ex)
             {
-                if (ex.Response.StatusCode == HttpStatusCode.Unauthorized)
-                {
-                    _logger.Error(ex, "API key is invalid: " + ex.Message);
-                    return new ValidationFailure("APIKey", "API key is invalid");
-                }
-
-                _logger.Error(ex, "Unable to send test message: " + ex.Message);
-                return new ValidationFailure("APIKey", "Unable to send test notification");
+                return new ValidationFailure("APIKey", ex.Message);
             }
             catch (Exception ex)
             {
-                _logger.Error(ex, "Unable to send test notification: " + ex.Message);
-                return new ValidationFailure("", "Unable to send test notification");
+                _logger.Error(ex, ex.Message);
+                return new ValidationFailure("", "Unable to send test notification. Check the log for more details.");
             }
         }
 
         private void ProcessNotification(StringDictionary message, NotifiarrSettings settings)
         {
             try
             {
-                var requestBuilder = new HttpRequestBuilder(URL).Post();
-                requestBuilder.AddFormParameter("api", settings.APIKey).Build();
+                var instanceName = _configFileProvider.InstanceName;
+                var requestBuilder = new HttpRequestBuilder(URL + "/api/v1/notification/prowlarr/").Post();
+                requestBuilder.AddFormParameter("instanceName", instanceName).Build();
+                requestBuilder.SetHeader("X-API-Key", settings.APIKey);
```

Comment:
This looks right but just to be safe, did you fire a test notification from your local build?

Reply:
tested works
also confirmed without trailing slash